app/controllers/api/v2/accredited_provider_training_providers_controller.rb

```diff
@@ -6,7 +6,21 @@ class AccreditedProviderTrainingProvidersController < API::V2::ApplicationContro
 
       def index
         authorize @provider, :can_list_training_providers?
-        providers = @provider.training_providers
+        if params[:filter]
```

Comment:
Not overly satisfied with this section of code. 
In order to use the CourseSearchService in all it's glory we need to pass in an ActiveRecordRelation containing courses ...

Reply:
does `Course.where(provider: provider.training_providers)` give you the same result? it should save the ruby `map`

Reply:
personal preference for `providers = if params[:filter]` and delete the line above. but don't know if rubocop allows it?

Reply:
Ah of course. Good spot. That will clean up the code nicely